src/nyx/phase2.cpp

```diff
@@ -168,6 +168,52 @@ namespace Nyxus
 		return true;
 	}
 
+#ifdef WITH_PYTHON_H
+	bool scanTrivialRoisInMemory (const std::vector<int>& batch_labels, const py::array_t<unsigned int>& intens_images, const py::array_t<unsigned int>& label_images, int start_idx)

```

Comment:
Since, we are using pointers to access the data, would it be better to add `py::array::c_style` as a template parameter of `py::array_t`, so that the contiguousness of the data buffer is guaranteed?

Reply:
Yes, thats a good point. I'll add in the c_style template parameter